### ISSUE TITLE
[BUG]When drawing the table, the "-" symbol in "|---|---|" grows infinitely.

### ISSUE THREAD

**Issue by AIlifuk** [ganesshkumar/obsidian-table-editor#35]
In actual use, due to the use of plug-ins, a total of 20 updates have been made, and the "-" symbol has become very long. Difficult to read in source mode。

**Comment by AIlifuk** [comment]
In another plugin "Advanced Tables", the same problem occurs, but when "Pad cell width using spaces" is turned off, there is no "-" sign growth problem.
May be the same problem, hope this problem can be fixed, or it can be temporarily solved by turning it off in settings.

**Comment by ganesshkumar** [comment]
Makes sense. I shall check the behavior in 'Advanced Tables' for this behavior.
